.claude/commands/map-check.md

```
@@ -0,0 +1,444 @@
+# Check each subtask
+for ST in $SUBTASKS; do
+  PENDING=$(jq -r ".pending_steps[\"$ST\"] | length" "$STATE_FILE")
+  if [[ "$PENDING" -gt 0 ]]; then
+    echo "❌ $ST has pending steps:"
+    jq -r ".pending_steps[\"$ST\"][]" "$STATE_FILE"
+  fi
+done
+```
+
+**If any subtask has pending steps:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)